internal/provider/resource_saml_idp.go

```diff
@@ -40,18 +45,18 @@ func resourceWizSAMLIdP() *schema.Resource {
 			},
 			"use_provider_managed_roles": {
 				Type:        schema.TypeBool,
-				Description: "Use provider managed roles?",
-				Optional:    true,
-				Default:     false,
+				Description: "Use provider (IdP) managed roles? This attribute needs to be set to accomodate for update lifecycle operations.",
+				Required:    true,
 			},
 			"allow_manual_role_override": {
 				Type:        schema.TypeBool,
-				Description: "Allow manual override for role assignment? Must be set `true` if `use_provided_roles` is false.",
-				Optional:    true,
-				Default:     true,
-				RequiredWith: []string{
-					"use_provider_managed_roles",
-				},
```

Comment:
Though the GraphQL schema labels this as required, this should remain optional with a default of true. Default values cannot be defined for required attributes. Changing the default behavior at this stage may have unintended consequences for users of the provider.